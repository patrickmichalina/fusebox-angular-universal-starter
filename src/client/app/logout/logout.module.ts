import { LogoutRoutingModule } from './logout-routing.module';
import { LogoutComponent } from './logout.component';
import { NgModule } from '@angular/core';
import { SharedModule } from '../shared/shared.module';

@NgModule({
  imports: [LogoutRoutingModule, SharedModule],
  declarations: [LogoutComponent],
  exports: [LogoutComponent]
})
export class LogoutModule { }
