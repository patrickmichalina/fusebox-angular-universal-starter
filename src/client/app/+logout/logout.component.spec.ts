import { LogoutComponent } from './logout.component';
import { async, TestBed, ComponentFixture } from '@angular/core/testing';

describe(LogoutComponent.name, () => {
  let fixture: ComponentFixture<LogoutComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [LogoutComponent]
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(LogoutComponent);
    fixture.detectChanges();
  })

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  })

  it('should compile', async(() => {
    expect(fixture.nativeElement).toBeTruthy();
  }));
});