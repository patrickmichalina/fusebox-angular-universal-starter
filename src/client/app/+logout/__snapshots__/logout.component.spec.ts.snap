// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutComponent should compile 1`] = `
<pm-logout>
  You have been logged out. Come back soon!
</pm-logout>
`;
